receiver/hostmetricsreceiver/README.md

```diff
@@ -71,8 +71,12 @@ network:
   <include|exclude>:
     interfaces: [ <interface name>, ... ]
     match_type: <strict|regexp>
+  scrape_conntrack: <true|false> # default = false
```

Comment:
I'm not sure what's the best option as well, especially considering the fact that I wasn't able to find any alternative of conntrack for Windows/ macOS. Is a seperate scraper a cleaner solution in case this is only relevant for Linux?

Reply:
The `processes`scraper seems to be Linux only so `conntrack` wouldn't be the first scraper lacking cross-platform support